qa/standalone/ceph-helpers.sh

```diff
@@ -131,6 +133,7 @@ function setup() {
     teardown $dir || return 1
     mkdir -p $dir
     mkdir -p $(get_asok_dir)
+    sudo sysctl -w ${KERNCORE}="core."${COREPATTERN}
```

Comment:
Yes, I didn't put this here because as @tchaikov says, teuthology sets core_pattern its own way.